pypxe/http.py

```diff
@@ -29,18 +30,29 @@ def __init__(self, **serverSettings):
         os.chdir (self.netbootDirectory)
         os.chroot ('.')
 
-        if self.mode_debug:
-            print 'NOTICE: HTTP server started in debug mode. HTTP server is using the following:'
-            print '\tHTTP Server IP: {}'.format(self.ip)
-            print '\tHTTP Server Port: {}'.format(self.port)
-            print '\tHTTP Network Boot Directory: {}'.format(self.netbootDirectory)
+        if self.logger == None:
+            import logging
+            import logging.handlers
+            # setup logger
+            self.logger = logging.getLogger("http")
+            handler = logging.StreamHandler()
+            formatter = logging.Formatter('%(asctime)s %(levelname)s %(message)s')
+            handler.setFormatter(formatter)
+            self.logger.addHandler(handler)
+
+            if self.mode_debug:
+                self.logger.setLevel(logging.DEBUG)
+
+        self.logger.debug('NOTICE: HTTP server started in debug mode. HTTP server is using the following:')
+        self.logger.debug('\tHTTP Server IP: {}'.format(self.ip))
+        self.logger.debug('\tHTTP Server Port: {}'.format(self.port))
+        self.logger.debug('\tHTTP Network Boot Directory: {}'.format(self.netbootDirectory))
```

Comment:
In L47-L49 should we remove the `\t` characters since we're using the logger now? If using the logger perhaps we shouldn't use the `\t`? Thoughts?